bin/disco_ssh.py

```diff
@@ -13,6 +13,7 @@
      -h --help              Show this screen
      --debug                Log in debug level
      --env ENV              Environment to operate in
+     --first                In case of multiple matching instances, connect to the first instead of failing
```

Comment:
how about --any? there is really no ordering in picking right?
(maybe you will have --earliest, --latest)